app/src/main/java/com/cornellappdev/android/eatery/presenter/MainPresenter.java

```diff
@@ -0,0 +1,75 @@
+package com.cornellappdev.android.eatery.presenter;
+
+import android.util.Log;
+
+import com.cornellappdev.android.eatery.Repository;
+import com.cornellappdev.android.eatery.model.EateryBaseModel;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashSet;
+
+public class MainPresenter{
```

Comment:
`MainPresenter {`, you're missing the space

Reply:
linting took care of it!